reconcile/ocm_clusters.py

```diff
@@ -18,17 +18,50 @@
 QONTRACT_INTEGRATION = "ocm-clusters"
 
 
+def _set_rosa_ocm_attrs(cluster: Mapping[str, Any]):
+    """Cluster account (aws) attribute from App-interface differs from the OCMSpec.
+    App-interface's account includes the details for all the OCM environments
+    but the cluster only needs the target OCM environment where it belongs.
+    This method changes the cluster dicctionary to include just those.
+    """
+    cluster_ocm_env_name = cluster["ocm"]["name"]
+
+    rosa = cluster["spec"]["account"]["rosa"]
+    ocm_env_filter = filter(
```

Comment:
Any particular reason you choose filter over list comprehension? I find list comprehensions easier to read and it's more common in qontract-reconcile.

Reply:
You're right. I don't have any strong preference for one over the other.